package.json

```diff
@@ -30,7 +30,7 @@
 		"compression": "^1.7.4",
 		"express": "^4.17.1",
 		"node-fetch": "^2.6.0",
-		"sirv": "^0.4.6"
+		"sirv": "^1.0.0-next.5"
```

Comment:
any reason you switched to the pre-release version?

Reply:
I expected types to be available in this version but they won't be until the next publish (hopefully).

Reply:
Reverted in newest commit